src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowMultiBranchProject.java

```diff
@@ -56,20 +57,27 @@
 
     private static final Logger LOGGER = Logger.getLogger(WorkflowMultiBranchProject.class.getName());
 
-    public WorkflowMultiBranchProject(ItemGroup parent, String name) {
+    @DataBoundConstructor
```

Comment:
Hello @jglick: would it be possible to elaborate on how it should be handled instead?